### PR TITLE
Doc sidebar

### PR DESCRIPTION
**Pull request by JoeyZwicker**

### CODE REVIEW

File: doc/development/analyze_your_data.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Analyze Your Data
+=================
+
+This section of documentation covers everything you'll need to know to deploy a working Pachyderm cluster and build your own analysis to process whatever data you want.
```
Comment (jdoliner) [inline doc/development/analyze_your_data.rst]: This seems kind of weird to me. This doc is going to cover everything you need to know to deploy which is already a separate doc. I think it's probably better to just link to that doc.

Also how is this doc going to be different from `getting_started.txt` and `beginner_tutorial.rst` it seems like there's going to be a lot of overlap with those.

File: doc/development/deploying_on_the_cloud.md [hunk 2]
```diff
@@ -1,61 +1,20 @@
...
-If you have docker running, you can run kubernetes right off of docker. If you don't, consider checking out:
-
-- minikube (TODO / link)
-- or use one of the [Deployment Options](./deploying.md) to host the Docker daemon.
-
-## Prerequisites
-
-- [Docker](https://docs.docker.com/engine/installation) >= 1.10
-
-## Port Forwarding
```
Comment (jdoliner) [inline doc/development/deploying_on_the_cloud.md]: This document is probably going to need a section on Port Forwarding or using `ADDRESS` to get `pachctl` talking to the deployment.

File: doc/_build/html/_sources/getting_started.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Getting Started
+===============
+If you've never used Pachyderm before, check out our :doc:`installation` instructions and :doc:`beginner_tutorial`. 
+
+
+Already got a kubernetes cluster? Deploy Pachyderm in one line:
+
+``$ kubectl create -f https://pachyderm.io/manifest.json``
+
+(You'll still need to install :doc:`pachctl`)
```
Comment (jdoliner) [inline doc/_build/html/_sources/getting_started.txt]: This file probably shouldn't be committed right?

**Comment by jdoliner** [comment]
Fixes #757.

**Comment by msteffen** [comment]
CI is currently failing, but I believe this change is safe to merge anyway, as all of these changes are docs changes (.md and .rst) except for assets_test (in which only two strings are changed). I believe these failures are also already well-understood and are being fixed in https://github.com/pachyderm/pachyderm/pull/860

Because of this, we are planning to submit this change. If submission causes problems, I believe they will be easy to fix.

mr bean

**Pull request merged**